### DISCUSSION TITLE
Option to manually read/write exist account details

### DISCUSSION THREAD

**Discussion by Lavoltz** [ente-io/ente#1034] [Enhancements]
Sometimes account details(name,private key,etc) may change. therefore a method to modify them could be useful.
(like 2FAS Auth)
![IMG_7643](https://github.com/ente-io/auth/assets/42258721/64f1595e-8b2a-4a06-9273-a55c65e4f2cb)
![IMG_7641](https://github.com/ente-io/auth/assets/42258721/11a34377-d10f-498b-a0b5-ea583562d89a)
![IMG_7642](https://github.com/ente-io/auth/assets/42258721/17c14d3a-f174-43a1-b21a-66de78c87cea)

**Comment by vishnukvmd** [comment]
Hey, if you click on the `+` icon at the bottom right, you will get an option to enter these details manually.

Please re-open the issue if you were looking for something else!

**Comment by Lavoltz** [comment]
> Hey, if you click on the `+` icon at the bottom right, you will get an option to enter these details manually.
> 
> Please re-open the issue if you were looking for something else!

Sorry for interrupt, i've just added some details, could you please re-open it? thx

**Comment by vishnukvmd** [comment]
Hey, could you please let us know which app this is?

**Comment by vishnukvmd** [comment]
Just to be clear, you can change your account name, secret, etc., by swiping left on the code. Is this what you were looking for?

**Comment by Lavoltz** [comment]
> Just to be clear, you can change your account name, secret, etc., by swiping left on the code. Is this what you were looking for?

Generally speaking, that's right. btw, sometimes copy secrets (from the "edit" form) to other 2fa apps is more convenient (and maybe safer) than export the whole plain-text backup file and copy the secret of alone

> Hey, could you please let us know which app this is?

It's called 2fas auth, the ios version, open-source in twofas/2fas-ios

**Comment by vishnukvmd** [comment]
> btw, sometimes copy secrets (from the "edit" form) to other 2fa app

You can do that here as well?

**Comment by Lavoltz** [comment]
> > btw, sometimes copy secrets (from the "edit" form) to other 2fa app
> 
> You can do that here as well?

What does it mean? If I understood it correctly - in the app 2fas, it's possible to "copy secrets (from the "edit" form of 2fas) to other 2fa apps"
